api/app/Http/Requests/Api/EventType/LocationTypes.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests\Api\EventType;
+
+final class LocationTypes
+{
+    private const LOCATION = 'location';
```

Comment:
It is better to name it `address`, as the user is entering the address, and additionally, we store the location.